kubernetes/helm/pinot/Chart.yaml

```diff
@@ -21,7 +21,7 @@ apiVersion: v1
 appVersion: 0.2.6-SNAPSHOT
 name: pinot
 description: Apache Pinot is a realtime distributed OLAP datastore, which is used to deliver scalable real time analytics with low latency. It can ingest data from offline data sources (such as Hadoop and flat files) as well as online sources (such as Kafka). Pinot is designed to scale horizontally.
-version: 0.2.6-SNAPSHOT
+version: 0.11.0
```

Comment:
this is just helmChart version, it's not sync'ed with pinot. you can update the default value from the `values.yaml`

Reply:
agreed. however, this is to make sure that some one doesn't end up deploying an older un-secure version of pinot by mistake if they don't provide a values.yaml which can happen while testing ci/cd workflows.

Reply:
i didn't understand what you meant. i've updated the pull request now.